### DISCUSSION TITLE
BanyanDB data

### DISCUSSION THREAD

**Discussion by SooryaMuthuraj** [apache/skywalking#11907] [General]
Hii All,

Can anyone explain where does banyanDB stores data? As in Elastic the data is stored in data folder of elastic root folder. We just need to know where does the same or similar thing happens with banyanDB.

**Comment by wu-sheng** [comment]
BanyanDB is not open for product use. It is dev stage only.
If you are interested, follow the concepts in the docs, especially reading codes.

**Reply by SooryaMuthuraj** [reply]
Then why you guys provided the Binary files in the Download section.
![image](https://github.com/apache/skywalking/assets/154515976/5e5aae3c-947d-4a40-b902-3a3d9fd13881)

**Reply by wu-sheng** [reply]
Binaries mean released.

> BanyanDB is going to be our next generation storage solution. It is still in alpha stage.

Read official doc declaration. https://skywalking.apache.org/docs/main/latest/en/setup/backend/backend-storage/

**Comment by hanahmily** [comment]
By default, data is stored in "/tmp", but you can change the path by using arguments. Refer to "banyand-server standalone -h" for more information.

**Reply by SooryaMuthuraj** [reply]
Please tell how to change the path

**Reply by CodePrometheus** [reply]
@surya55muthuraj I think you can try this `banyand-server standalone --metadata-root-path=""`, the configuration of `stream-root-path` and `measure-root-path` is the same.

**Reply by SooryaMuthuraj** [reply]
Thanks
